lua/starfall/libs_sv/entities.lua

```diff
@@ -548,6 +548,14 @@ function ents_methods:isPlayerHolding()
 	return ent:IsPlayerHolding()
 end
 
+---Returns if the entity is a constraint.
+-- @server
+-- @return Boolean if the entity is a constraint
+function ents_methods:isConstraint()
+	local ent = getent(self)
```

Comment:
My only suggestion for this one is to make it `return getent(self):IsConstraint()`